Apps/phredmine/readme.html

```diff
@@ -0,0 +1,21 @@
+<!-- File: readme.html
+  Copyright (c) 2021 Splunk Inc.
+
+  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+-->
+
+<p>
+  <h2>Redmine</h2>
+</p>
+
+<p>Redmine is an open source ticketing system, and the actions performed by these API calls depend to some extend on the configuration on the Redmine instance.</p>
+
+<p>
+<b>The functioning of On Poll</b>
+    <li>The On Poll action works in 2 steps. In the first step, all the tickets (issues) in a defined time duration will be fetched. In the second step, all the components (e.g. fields) of the tickets (retrieved in the first step) will be fetched. A container will be created for each ticket and for each ticket all the components will be created as the respective artifacts.</li>
```

Comment:
```suggestion
    <li>The On Poll action works in 2 steps. In the first step, all the tickets (issues) in a defined time duration will be fetched. In the second step, all the components (e.g. fields) of the tickets (retrieved in the first step) will be fetched. A container will be created for each ticket and for each ticket all the components will be created as the respective artifacts.</li>
```
The On Poll action works in 2 steps. In the first step, all the tickets (issues) will be fetched in defined time duration. In the second step, all the components (e.g. fields) of the tickets (retrieved in the first step) will be fetched. A container will be created for each ticket and for each ticket all the components will be created as the respective artifacts.